frontend/src/components/InfoBox/InfoBoxTags.js

```diff
@@ -139,7 +101,7 @@ class Tags extends Component {
               title={
                 <span>
                   {courseName} is <b>{this.props.credits}</b> credit unit
-                  {this.props.credits === 1 || 's'}.
+                  {this.props.credits === 1 || 's'}
```

Comment:
I feel like it'd be good to always return a string here:
```suggestion
                  {this.props.credits === 1 ? '' : 's'}
```